client/app/components/packages/pas-form/edit.js

```diff
@@ -7,6 +7,10 @@ import SaveablePasFormValidations from '../../../validations/saveable-pas-form';
 import SubmittablePasFormValidations from '../../../validations/submittable-pas-form';
 
 export default class PasFormComponent extends Component {
+  @tracked saveError;
+
+  @tracked submitError;
+
```

Comment:
My worry here is that this is more state that we have to clean up. What happens when they fix the error? Does the message persist?

Reply:
that's a good point. I'm going to look into that API for dealing with errors on ember data and try to refactor :)